src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs

```diff
@@ -447,6 +447,8 @@ public void Abort(QuicAbortDirection abortDirection, long errorCode)
             return;
         }
 
+        ThrowHelper.ValidateErrorCode(nameof(Abort), errorCode);
```

Comment:
what about the compiler-filled parameter `propertyName`, won't it produce weird error message? Same for QuicConnection.CloseAsync